### DISCUSSION TITLE
isNaN

### DISCUSSION THREAD

**Discussion by rvision** [LegendApp/legend-state#149] [General]
From the docs:

> isNaN is slow: This surprised us, but isNaN was causing significant slowdown. +n - +n < 0 is a much faster way to check if a string is a number.

shouldn't this detection code look like:

+idx - +idx === 0

?

**Comment by jmeistrich** [comment]
Oh good catch! A small typo there, should be 1 instead of 0. We actually don't even use it in our code anymore 😂. We've updated the docs to fix it. Thank you!
